framework/src/main/java/org/checkerframework/common/returnsrcvr/qual/This.java

```diff
@@ -0,0 +1,13 @@
+package org.checkerframework.common.returnsrcvr.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.PolymorphicQualifier;
+
+/** This represent methods that returns {@code this} */
```

Comment:
The `@This` annotation does not represent a method.  Please rephrase.

Nit: please end sentences with punctuation.